src/Frontend/src/components/failedmessages/EditRetryDialog2.vue

```diff
@@ -122,6 +108,11 @@ async function retryEditedMessage() {
 }
 
 function initializeMessageBodyAndHeaders() {
+  function getHeaderValue(key: string) {
+    const header = local.headers.find((header: HeaderWithEditing) => header.key === key);
```

Comment:
i realise this works because of hoisting, but this function is declared before `local` is. 
```suggestion
    const header = headers.value.find((header: HeaderWithEditing) => header.key === key);
```